#! /bin/sh /usr/share/dpatch/dpatch-run
## 29_fix_mythwelcome.dpatch by Michael Haas  <laga@laga.ath.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Backport changeset 18671 from trunk
## DP: commit message: http://svn.mythtv.org/trac/changeset/18671
## DP: original bug report: http://svn.mythtv.org/trac/changeset/18671
## DP: original patch by alrcan (at) blueyonder.co.uk

@DPATCH@
diff -urNad mythtv-fixes~/programs/mythwelcome/welcomedialog.cpp mythtv-fixes/programs/mythwelcome/welcomedialog.cpp
--- mythtv-fixes~/programs/mythwelcome/welcomedialog.cpp	2008-10-15 23:57:21.000000000 +0200
+++ mythtv-fixes/programs/mythwelcome/welcomedialog.cpp	2008-11-20 09:27:56.000000000 +0100
@@ -102,7 +102,7 @@
     bool bAutoStartFrontend = gContext->GetNumSetting("AutoStartFrontend", 1);
 
     if (state == 1 && bAutoStartFrontend)
-        startFrontend();
+        QTimer::singleShot(500, this, SLOT(startFrontend()));
 
     // update status now
     updateAll();
